docusaurus.config.js

```diff
@@ -93,19 +93,20 @@ const config = {
       footer: {
         style: 'dark',
         links: [
-          {             
+          {
             label: 'Change cookie preferences',
             href: '/cookie-preferences',
           },
           {
             label: 'Terms and conditions',
-            href: '/terms-and-conditions', 
+            href: '/terms-and-conditions',
           }
         ],
         copyright: `Copyright © ${new Date().getFullYear()} Snowplow Analytics Ltd. Built with Docusaurus.`,
       },
       prism: {
         theme: require('prism-react-renderer/themes/dracula'),
+        additionalLanguages: ['arduino', 'csharp', 'docker', 'gradle', 'java', 'php', 'properties', 'r', 'ruby', 'scala', 'swift'],
```

Comment:
I think we should preemptively list all languages we use (`git grep -Eho '```[a-z]+' '*.md' | sort | uniq | sed 's/```//'`), so that we don’t depend on what Prism deems to be good defaults.

Reply:
You can't - build fails if you add language X and X isn't supported by Prism.  Gives error like this for `html`

```
Error: Cannot find module './prism-html'
    at Array.forEach (<anonymous>)
[ERROR] Unable to build website for locale en.

● Client █████████████████████████ cache (99%) shutdown IdleFileCachePlugin
 resolve build dependencies

● Server █████████████████████████ cache (99%) shutdown IdleFileCachePlugin
 process pending cache items

[ERROR] Error: Failed to compile with errors.
    at /Users/andyhay/code/snowplow.github.io/node_modules/@docusaurus/core/lib/webpack/utils.js:180:24
    at /Users/andyhay/code/snowplow.github.io/node_modules/webpack/lib/MultiCompiler.js:554:14
    at processQueueWorker (/Users/andyhay/code/snowplow.github.io/node_modules/webpack/lib/MultiCompiler.js:491:6)
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)
[INFO] Docusaurus version: 2.0.1
```

Reply:
That’s annoying :(

Reply:
In this case, keep it as you did, but add a comment explaining that we can’t add more (and a link to where the default ones are defined)

Reply:
Reverted other changes, just single change in config file with comments referencing the links and warning about doing a build to check (yarn start just fails with no displayed error)